### ISSUE TITLE
repeat output = model(data) in recognition.py 

### ISSUE THREAD

**Issue by mmchen88** [open-mmlab/mmskeleton#439]
for data, label in data_loader:
        with torch.no_grad():
            output = model(data)
            output = model(data).data.cpu().numpy()
